src/lib/child_processes/README.md

```diff
@@ -0,0 +1,21 @@
+# Child processes
```

Comment:
Can you add a code directory to this README.md which describes what the various files in this library are for?

Reply:
Can you also talk more about how this system handles the lifecycle of child processes? As in, how is stdin/stdout managed, what does it do when a child dies, what does it do when it receives certain signals, etc.